xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -1063,4 +1076,40 @@ public JobLog toRestJobLog(Collection<LogEvent> logs, URI self)
 
         return log;
     }
+
+    /**
+     * check if the given property should be exposed via REST
+     * @param restProperty the property to be read/written
+     * @param forWrite true iff we want to write to the XObject property
+     * @return true if the property is considered public for read or write
+     */
+    private boolean isTypePublic(Property restProperty, boolean forWrite)
```

Comment:
s/isTypePublic/isPublicType ?

Reply:
yep, I wondered, too, if the other way round would be better. will change this.